### PR TITLE
SAK-49135 Drop Box: Pink error banner displays when selecting Upload files to multiple Drop Box folders tab

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-49135

### CODE REVIEW

File: content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm [hunk 1]
```diff
@@ -50,7 +50,7 @@
                     (<a href="#" onclick="openMoreInfoCopyrightWindow('copyright');" title="[$tlang.getString('cright.newwin')]"><span class="skip"> [$tlang.getString('cright.newwin')]</span>$tlang.getString('cright.fairuse')</a>)
                 </small>
             </div>
-            <div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="show"#else class="hide"#end>
+            <div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="d-none"#end>
```
Comment (adrianfish) [inline content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm:53]: This doesn't look right. Should it not be != ?

File: content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm [hunk 2]
```diff
@@ -274,7 +274,7 @@ button.btn.btn-default.multiselect-clear-filter{
 				(<a href="#" onclick="openMoreInfoCopyrightWindow('copyright');" title="[$tlang.getString('cright.newwin')]"><span class="skip"> [$tlang.getString('cright.newwin')]</span>$tlang.getString('cright.fairuse')</a>)
 			</small>
 		</p>
-		<div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="form-group show"#else class="hide"#end>
+		<div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="form-group show"#else class="d-none"#end>
```
Comment (adrianfish) [inline content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm:277]:
```suggestion
		<div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="form-group"#else class="d-none"#end>
```

**Review by adrianfish — CHANGES_REQUESTED** [review]
Couple of odd things here and there.

File: content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm (hunk 2 above)
Comment (kunaljaykam) [reply content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm:277]: The JS needs it I think.

File: content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm (hunk 1 above)
Comment (kunaljaykam) [reply content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm]: Good catch.

**Review by adrianfish — APPROVED** [review]
Thanks Kunal :)

**Pull request merged**